Makefile

```diff
@@ -810,11 +810,11 @@ proxy-mmlogic: build/toolchain/bin/kubectl$(EXE_EXTENSION)
 	@echo "MmLogic Trace: http://localhost:$(MMLOGIC_PORT)/debug/tracez"
 	$(KUBECTL) port-forward --namespace $(OPEN_MATCH_KUBERNETES_NAMESPACE) $(shell $(KUBECTL) get pod --namespace $(OPEN_MATCH_KUBERNETES_NAMESPACE) --selector="app=open-match,component=mmlogic,release=$(OPEN_MATCH_CHART_NAME)" --output jsonpath='{.items[0].metadata.name}') $(MMLOGIC_PORT):51503 $(PORT_FORWARD_ADDRESS_FLAG)
 
-proxy-evaluator: build/toolchain/bin/kubectl$(EXE_EXTENSION)
```

Comment:
I think we can actually get rid of the proxy for synchronizer as it is an internal service

Reply:
Yes, as per offline discussion we had, this component may have some differences such as not needing http proxy (and the proto, pb being under internal etc.) Let me keep this PR to just renaming and handle all those in a separate issue: https://github.com/googleforgames/open-match/issues/534